Modelica/package.mo

```diff
@@ -2278,7 +2278,7 @@ Graphical illustrations shall not be added in the diagram layer, but can be adde
 </html>"));
     end Icons;
     annotation (DocumentationClass=true,Documentation(info="<html>
```

Comment:
(Initiating conversation to line above where it really belongs.)

In view of https://github.com/modelica/ModelicaStandardLibrary/blob/fa90ce72720e3ffee385d1f5ab6081eb43637a94/ModelicaReference/package.mo#L1372C1-L1372C1,

maybe we should align and say _top-level package_ instead of _main package_?

Reply:
[`Modelica.UsersGuide.Overview`](https://github.com/modelica/ModelicaStandardLibrary/blob/fa90ce72720e3ffee385d1f5ab6081eb43637a94/Modelica/package.mo#L13) even says "The Modelica Standard Library consists of the following main sub-libraries..."
Meaning sub-libraries being `Blocks`, etc.

And we have "Library officers" being in charge with specific sub-libraries.

So IMO the wording is quite confusing.

Reply:
My assumption is that main package meant top-level user-facing package, but apart from 'main package' not being defined I don't see why we need to specify 'top-level' or 'main' at all.

Current:

User-facing main packages of the Modelica Standard Library (such as Modelica and Complex) should be compliant with the UsersGuide stated in this documentation:

vs.

User-facing packages of the Modelica Standard Library (such as Modelica and Complex) should be compliant with the UsersGuide stated in this documentation:

Or perhaps rephrase:

Packages of the Modelica Standard Library that are user-facing (such as Modelica and Complex) should be compliant with the UsersGuide stated in this documentation:

Reply:
To my understanding so far (disregarding the documentation): *Modelica*, *Complex*, etc., are all Modelica libraries containing sub-packages. But I'm willing to re-think here.

Btw: Better is "such as Modelica **or** Complex"

Reply:
> To my understanding so far (disregarding the documentation): _Modelica_, _Complex_, etc., are all Modelica libraries containing sub-packages. But I'm willing to re-think here.

I would say that they are both libraries and packages. I agree that there's room for a clearer statement without knowing the best one, but at least I don't think we should include top-level or main in that text.

> Btw: Better is "such as Modelica **or** Complex"

Agreed.

Reply:
>  I don't think we should include top-level or main in that text.

Fine to me.

Reply:
OK, neither _main_ nor _top-level_ is used now.  I'll resolve and re-start the discussion based on this.